### PR TITLE
3.1.1

### PR DESCRIPTION
**Pull request by nickpatrick**
- Deserialize isochrone geofence geometry
- Expose matrix API
- Expose `truck` and `motorbike` travel modes
- Clean up docs links

### CODE REVIEW

**Review by jsani-radar — APPROVED** [review]

**Comment by nickpatrick** [comment]
Added:
- Expose matrix API
- Expose `truck` and `motor_scooter` travel modes
@tjulien @ahwreck @brettguenther

File: RadarSDK/Radar.m [hunk 1]
```diff
@@ -640,6 +660,12 @@ + (NSString *)stringForMode:(RadarRouteMode)mode {
     case RadarRouteModeCar:
         str = @"car";
         break;
+    case RadarRouteModeTruck:
+        str = @"truck";
+        break;
+    case RadarRouteModeMotorScooter:
+        str = @"motor_scooter";
```
Comment (tjulien) [inline RadarSDK/Radar.m:667]: right now this is `motorbike` in radar (which gets mapped to `motor_scooter` in valhalla)

https://github.com/radarlabs/server/pull/3118/files#diff-02d01ebe7d564e3772bd83bab5289d376d6c0e5781f9fea7be0d4e6156a579d9R169

https://valhalla.readthedocs.io/en/latest/api/turn-by-turn/api-reference/#costing-models

Comment (nickpatrick) [reply RadarSDK/Radar.m:667]: Good catch, will swap

**Comment by jsani-radar** [comment]
since this repo is public, could we enable branch protection such that pushing subsequent commits requires a re-approval? trying to remove mine since there's been more recent changes, might as well get it to happen automatically /cc @tjulien

**Comment by nickpatrick** [comment]
@jsani-radar Good call, just turned this on.

**Comment by nickpatrick** [comment]
@jsani-radar @tjulien Ready for re-review.

**Comment by tjulien** [comment]
@jsani-radar @nickpatrick The best way to achieve this is to have a clubhouse + PR per feature / bug-fix.  Either branched off of `master` or `3.1.1.`. Next time around.

**Review by tjulien — APPROVED** [review]

**Comment by tjulien** [comment]
https://app.clubhouse.io/radarlabs/story/5778/expose-distance-matrix-api-and-truck-motorbike-travel-modes-in-ios-sdk

**Pull request merged**